codegen/src/main/java/software/amazon/awssdk/codegen/poet/client/traits/NoneAuthTypeRequestTrait.java

```diff
@@ -26,6 +26,9 @@
  * Trait which defines if a given request needs to be authenticated.
  * A request is not authenticated only if it has "auththpe" trait  explicitly marked as "none"
  */
+// TODO(sra-identity-and-auth): I think this class can be deleted. Old clients will still have the writer for this generated
```

Comment:
I was a little hesitant to remove this and wanted someone to review if this end state is ok (code base would only have a reader for this attribute and no write). But I'm going to go ahead and remove this class in next revision. If anyone has concerns, let me know.